content/applications/productivity/discuss.rst

```diff
@@ -1,105 +1,145 @@
+checking their *status*. The status is shown on the right side of a contact's name on the sidebar,
+on the *messaging menu*, and when listed in the *Chatter*.
 
 - Green = online
 - Orange = away
 - White = offline
 - Airplane = out of the office
 
-.. image:: discuss/status.png
-   :height: 300
-   :alt: View of the contacts' status for Odoo Discuss.
+Delete a direct message conversation
+------------------------------------
+
+To delete direct message conversations, click the :guilabel:`X` next to the name in the
+:guilabel:`Direct Messages` section of the sidebar.
+
+.. note::
+  This does **not** delete the :guilabel:`Direct Messages` in the conversation. The direct message
+  conversation's history is visible when another direct message with the same person, or group, is
+  created.
+
+.. _discuss/meetings:
+
+Meetings
+========
+
+In **Discuss**, :guilabel:`Meetings` are video calls. To start a meeting from the **Discuss**
+application home page, click :guilabel:`Start a meeting` in the top-left corner, and select who to
+invite to the meeting, via the :guilabel:`Invite People` drop-down window that appears. To start a
+meeting from a direct message, click the :icon:`fa-phone` :guilabel:`Start a Call` icon in the
+top-right.
+
+.. image:: discuss/meeting.png
+   :alt: View of a Meeting in Odoo Discuss.
+
+Once a meeting has been started, the buttons on the bottom of the call screen can be used to
+:icon:`fa-microphone` :guilabel:`Mute/Unmute`, :icon:`fa-headphones` :guilabel:`Deafen/Undeafen`,
+and :icon:`fa-video-camera` :guilabel:`Turn camera on/off`. Clicking the :icon:`fa-ellipsis-v`
+:guilabel:`More` icon allows the user to :icon:`fa-hand-paper-o` :guilabel:`Raise Hand`,
+:icon:`fa-desktop` :guilabel:`Share Screen`, or :icon:`fa-arrows-alt` :guilabel:`Enter Full Screen`.
+
+.. _discuss_app/notification_preferences:
+
+Choose notifications preference
+===============================
+
+Access user-specific preferences for the **Discuss** app by navigating to :menuselection:`Settings
+app --> Manage Users`, select a user, and then click the :menuselection:`Preferences` tab.
+
+.. image:: discuss/preferences-user.png
+   :alt: View of the Preferences tab for Odoo Discuss.
+
+By default, the :guilabel:`Notification` field is set as :guilabel:`Handle by Emails`. With this
+setting enabled, a notification email is sent by Odoo every time a message is sent from the
+chatter, a note is sent with an `@` mention (from *Chatter*), or a notification is sent for a record
+that the user follows.
+
+By choosing :guilabel:`Handle in Odoo`, the above notifications are shown in the **Discuss** app's
+:guilabel:`Inbox`.
+
+Chat from different applications
+================================
+
+The **Discuss** application enables communication across all of Odoo's applications. To view chats
+and channels, or start a new message, select the speech bubbles in the upper-right corner of another
+app in Odoo.
```

Comment:
it might be better to reword this in a way similar to : "...select the speech bubbles that are constantly present in the upper-right corner of the database header..." or something along those lines -- because they are still present even when the user is not in an app.